sdk/dotnet/Sql/User.cs

```diff
@@ -151,6 +151,16 @@ namespace Pulumi.Gcp.Sql
     /// });
     /// ```
     /// 
+    /// ## Ephemeral Attributes Reference
+    /// 
+    /// The following write-only attributes are supported:
+    /// 
+    /// * `password_wo` - (Optional) The password for the user. Can be updated. For Postgres
```

Comment:
We cannot support write-only attributes fully yet. I think the safest thing here to do would be to patch them until we can either suppress them more effectively (e.g. via the bridge) or until https://github.com/pulumi/pulumi/issues/16600 is implemented fully.

Patching is unfortunately the messiest option, but it's what we did for AWS: https://github.com/pulumi/pulumi-aws/issues/5231